rllib/algorithms/bc/tests/test_bc_rl_module.py

```diff
@@ -0,0 +1,87 @@
+import os
+from pathlib import Path
+import unittest
+
+import ray
+import ray.rllib.algorithms.bc as bc
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.test_utils import (
+    check_compute_single_action,
+    check_train_results,
+    framework_iterator,
+)
+
+tf1, tf, tfv = try_import_tf()
+
+
+class TestBRLModuleC(unittest.TestCase):
```

Comment:
Can we consolidate this test with the other one that does not use RL Modules?
They should be the same (save for the rl module setting), so there should be a loop that loops over `rl_modules=True/False`. By doing that we can be sure/demonstrate that the task is the exact same so that the new implementation is equivalent. We also get the chance to mitigate code divergence between the old and new testing code in the near future.